src/css/colors.scss

```diff
@@ -1,16 +1,22 @@
 // Pink
-$xl-pink: #EE1A64;
+$xlPink: #EE1A64;
 
 // Yellow
-$default-yellow: #F6C14D;
+$defaultYellow: #F6C14D;
 
 // Gray
-$default-gray: #858585;
-$disabled-gray: #B6B6B6;
+$defaultGray: #858585;
+$grayTwo: #9B9B9B;
+$disabledGray: #B6B6B6;
 
 // Dark
 $black: #000000;
-$footer-container: #161616;
+$footerContainer: #161616;
```

Comment:
I think this should have a color name and not the name of the component where is used